core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -242,39 +237,46 @@ function (Environment $env, $context, $template, $variables = [], $withContext =
 
     public function getFilters(): array
     {
-        $escaperFilter = static function (Environment $env, $string, $strategy = 'html', $charset = null, $autoescape = false) {
+        $escaperFilter = function (Environment $env, $string, string $strategy = 'html', string|null $charset = null, bool $autoescape = false) {
+            $runtime = $this->environment->getRuntime(EscaperRuntime::class);
```

Comment:
```suggestion
        $escaperFilter = static function (Environment $env, $string, string $strategy = 'html', string|null $charset = null, bool $autoescape = false) {
            $runtime = $env->getRuntime(EscaperRuntime::class);
```
I think we should use the `Environment` that gets passed to the filter here.

Reply:
Changed in 6f7ccc2f4c60b862ab65cb0a8ff17ffc4432060c.